scripts/v2-migration/get-governance-rewards.js

```diff
@@ -0,0 +1,121 @@
+const { ethers, config } = require('hardhat');
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+
+const VERSION_DATA_URL = 'https://api.nexusmutual.io/version-data/data.json';
+const EVENTS_START_BLOCK = 0;
+const OUTPUT_FILE = path.join(
+  config.paths.root,
+  'scripts/v2-migration/output', // dir
+  'governance-rewards.json', // filename
+);
+
+const getContractFactory = async providerOrSigner => {
+  const data = await fetch(VERSION_DATA_URL).then(r => r.json());
+  const abis = data.mainnet.abis
+    .map(item => ({ ...item, abi: JSON.parse(item.contractAbi) }))
+    .reduce((data, item) => ({ ...data, [item.code]: item }), {});
+
+  return async code => {
+    const { abi, address } = abis[code];
+    return new ethers.Contract(address, abi, providerOrSigner);
+  };
+};
+
+function onlyUnique(value, index, self) {
+  return self.indexOf(value) === index;
+}
+
+const main = async (provider, useCache = true) => {
+  // check the cache first
+  if (useCache && fs.existsSync(OUTPUT_FILE)) {
+    console.log('Using cached data for Governance rewards');
+    return JSON.parse(fs.readFileSync(OUTPUT_FILE).toString());
+  }
+
+  const factory = await getContractFactory(provider);
+  const governance = await factory('GV');
+
+  console.log(`Fetching all vote events to extract addresses...`);
+  const filter = governance.filters.Vote();
+  filter.fromBlock = EVENTS_START_BLOCK;
+  const voteLogs = await provider.getLogs(filter);
+  const addresses = voteLogs
+    .map(log => {
+      const data = governance.interface.parseLog(log);
+      const { from } = data.args;
+      return from.toLowerCase();
+    })
+    .filter(onlyUnique);
+
+  // Computed using Dune: https://dune.com/queries/2012946
```

Comment:
would it be possible to do this query here dinamically?

Are we considering hardcoding here ok because delegations are disabled?

Reply:
Delegations are disabled, so this list won't change

Reply:
Also, we're using this list just for assertions in the test, we're not doing any magic txs

Reply:
cool that's fine then